Utils/azuretools-core/src/com/microsoft/azuretools/ui/embeddedbrowser/JxBrowserUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *   <p/>
+ *  All rights reserved.
+ *   <p/>
+ *  MIT License
+ *   <p/>
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ *  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ *  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ *  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *  <p/>
+ *  The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ *  the Software.
+ *   <p/>
+ *  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ *  THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ *  TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ */
+
+package com.microsoft.azuretools.ui.embeddedbrowser;
+
+import com.google.common.collect.ImmutableMap;
+
+public class JxBrowserUtil {
+    private static final ImmutableMap<String, String> versionMap = ImmutableMap.of(
```

Comment:
```
diff /home/jenkins/dependencyLog//develop/intellij/528-intellij.log /home/jenkins/dependencyLog//PR/intellij/1279-intellij.log
+ cat /home/jenkins/dependencyLog//patch/PR-1279.patch
+ wc -l
+ [ 5 -eq 0 ]
+ cat /home/jenkins/dependencyLog//patch/PR-1279.patch
135c135,136
< |    \--- com.google.code.gson:gson:2.7
---
> |    +--- com.google.code.gson:gson:2.7
> |    \--- com.google.guava:guava:20.0
+ echo ALERT: New dependencies introduced!
```

You could try Java `java.util.AbstractMap.SimpleImmutableEntry<K,V>`

Reply:
Use ImmutableMap for now

---
In reply to: [154283480](https://github.com/Microsoft/azure-tools-for-java/pull/1279#discussion_r154283480) [](ancestors = 154283480)